modules/data.land/R/extract_soil_nc.R

```diff
@@ -362,19 +319,16 @@ extract_soil_gssurgo <- function(outdir, lat, lon, size=1, grid_size=3, grid_spa
         return(NULL)
       })
     })
-  # removing the nulls or the ones that throw exception in the above trycatch
-  out.ense<- out.ense %>%
+  
+  # remove nulls 
+  out.ense <- out.ense %>%
     purrr::discard(is.null)
   
-  out.ense<-out.ense %>% 
+  out.ense <- out.ense %>% 
```

Comment:
An additional refactor that would make this read easier: Getting to the bottom of the function I notice there's a long series (at least lines 307-330) of assignments to `out.ense` that each perform one mutation -- this is a pipeline! Might want to consider writing it that way explicitly by changing the repeated `out.ense <- out.ense...` to `%>%`